tests/.env

```diff
@@ -0,0 +1,12 @@
+MANIFEST=warmup-manifest-release.yml
```

Comment:
the `.env` file should be an example and shouldn't be committed "as-is", instead it is best to call it `.env.example` for whoever using the repo to decide whether to use it or not (i.e. there might already be one in use)